index.html

```diff
@@ -135,8 +155,175 @@ <h1>2023.08.08(TUE) ~ 08(THU)</h1>
             <p class="end_title">
                 see the whole program
             </p>
+            <p class="global">
+              Join CC Global Summit 2015
+            </p>
+        </article>
+      </section>
+
+      <section class="speaker_section">
+        <article class="speaker_content">
+          <p class="title">featured speakers <span></span></p>
+          <div class="speakers">
+            <div class="speaker_grp1">
+              <!-- 
+              <div class="speaker">
+                <img src="./image/user1.png" alt="">
+                <div class="speaker_intro">
+                  <div class="user_title">
+                    <h1>Yochai Benkler</h1>
+                    <p>
+                      Berkman Professor of Entrepreneurial 
+                      Legal Studies at Harvard Law School
+                    </p>
+                    <span></span>
+                  </div>
+                  <p>
+                    Benkler studies commons-based peer 
+                    production, and published his seminal 
+                    book, The Wealth of Networks in 2006
+                  </p>
+                </div>
+              </div>
+
+              <div class="speaker">
+                <img src="./image/user2.png" alt="">
```

Comment:
Please ensure to remove unused pieces of code from your codebase in order to maintain readability and focus on what is working.